ci/build_wheel.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e -u -o pipefail
+
+./build.sh
+python -m build \
+    -no-build-isolation \
```

Comment:
```suggestion
    --no-build-isolation \
```

Reply:
and even this was wrong... it's `--no-isolation`, not `--no-build-isolation`, for `build. `--no-build-isolation` is a `pip` thing

just pushed a commit to fix that